R/class-KPartiteNetwork.R

```diff
@@ -1,82 +1,51 @@
-#' @include methods-KPartiteNetwork.R
-check_partitions <- function(object) {
-  errors <- character()
-  
-  if (!!length(getAllNodeIds(object))) {
-    # Ensure that no node is in multiple partitions
-    if (length(getAllNodeIds(object)) > data.table::uniqueN(getAllNodeIds(object))) {
-      errors <- c(errors, 'Found a node in multiple partitions. Nodes can only exist in one partition.')
+  isLinkWithinPartition <- function(link, partitions) {
```

Comment:
Suggestion to rename to "isIntraPartitionLink", becuase the field often calls links "inter-" vs "intra-" something.